database/migrations/2024_04_24_000002_update_passwords_field_to_be_nullable.php

```diff
@@ -24,7 +24,7 @@ public function down()
     {
         Schema::table('users', function (Blueprint $table) {
             $table->string('password')->nullable(false)->change();
-            $table->string('name')->nullable(false)->change();
+            // $table->string('name')->nullable(false)->change();
```

Comment:
@tnylea open to suggestions on how we should handle this here. As the down method will fail if there are already users will null names.

Reply:
Sorry, I didn't respond here. But looks like we got this resolved. so I'll resolve this ;)